src/models/shopping-cart-item.model.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Entity, model, property} from '@loopback/repository';
+
+/**
+ * Item in a shopping cart
+ */
+@model()
+export class ShoppingCartItem extends Entity {
+  /**
```

Comment:
Nitpick: the comments for the properties seem redundant, so I suggest removing them, or adding some more meat to them.